examples/application/sourcemap.py

```diff
@@ -0,0 +1,65 @@
+"""
+This example shows how to use the source map feature to for a PyTeal program.
+
+To enable the source mapping (at the time of this writing 3/13/2023)
+one must import `FeatureGate` from `feature_gates` and call `FeatureGate.set_sourcemap_enabled(True)`.
+This import must occur before any object from PyTeal is imported, as PyTeal will
+default to its own sourcemap configuration if not set beforehand.
+"""
+
+from feature_gates import FeatureGates
+
+
```

Comment:
wouldn't it be less confusing if pyteal imports and feature setting were at the top
like 
```
from feature_gates import FeatureGates
FeatureGates.verbose = True
FeatureGates.set_sourcemap_enabled(True)
import pyteal
```

Reply:
63ada3df475de1213dbc55aca6a8713ffc03a98c